Makefile

```diff
@@ -6,26 +6,54 @@
 # This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
 # distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
-# date: 2015-10-08
+# date: 2015-10-11
 #
 
 #-----------------------------------------------------------------------------------------------------------------------
+# make control/macros
+#-----------------------------------------------------------------------------------------------------------------------
+# defaults
+mk_do_inc := 1
+
+mk_simple_targets := clean configure distclean menuconfig help
+
+# This macro checks, if the make target list MAKECMDGOALS contains the given simple target $1. If so, it
+# executes mk_set_simple_target to set/clear some variables.
```

Comment:
Please use line length of 120 characters, so yes - extend the comments that you added. If you need second line anyway - no problem, let's just keep all the lines in the Makefile similar.